client/packages/requisitions/src/RequestRequisition/ListView/ProgramRequisitionOptions.tsx

```diff
@@ -203,6 +203,20 @@ export const ProgramRequisitionOptions = ({
       <LabelAndOptions {...suppliers} optionKey="name" />
       <LabelAndOptions {...orderTypes} optionKey="name" />
       <LabelAndOptions {...periods} optionKey="name" />
```

Comment:
This periods one should be removed right?

Reply:
Opps forgot to remove~ good catch [a56520c](https://github.com/msupply-foundation/open-msupply/pull/6357/commits/a56520ccef709b30daf522ee33f3f1495769927e)